### PR TITLE
docs: add kind video

### PR DESCRIPTION
**Pull request by cdrage**
docs: add kind video

### What does this PR do?

* Adds new section regarding "installing" the extension
* Talks that it is built-in
* Adds a tutorial video on cluster creation

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop-internal/issues/349

### How to test this PR?

View the video / page

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by cdrage** [comment]
https://github.com/containers/podman-desktop-media/pull/24 must go in first before merging this PR

**Review by Firewall — APPROVED** [review]

**Comment by cdrage** [comment]
Link updated and ready for review @shipsing

File: website/docs/kind/installing-extension.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Kind is a *built-in* extension to Podman Desktop and no installation is necessary.
+
+:::
+
+The Kind extension provides the capability of creating a local Kubernetes cluster using only containers rather than a Virtual Machine. 
+
+#### Procedure
+
+Simply [install Podman Desktop](/docs/installation) to have the extension available. To confirm the built-in extension is installed, Kind can be found within the **Extensions** section of Podman Desktop.
```
Comment (shipsing) [inline website/docs/kind/installing-extension.md]:
```suggestion
[Install Podman Desktop](/docs/installation) to have the extension available. To confirm the built-in extension is installed, you can find Kind within the **Extensions** section of Podman Desktop.
```

**Comment by shipsing** [comment]
@cdrage, Just one suggestion. rest looks good.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**